client/src/components/Input.js

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+import COLORS from '../utils/COLORS';
+
+const TextInput = styled.input`
+  background-color: ${COLORS.primaryInputBackground};
+  color: ${COLORS.primaryFont};
+  font-size: 20px;
+  border-radius: 14px;
+  width: ${(props) => props.width};
+  height: ${(props) => props.height};
+  padding-left: 15px;
+  border: none;
```

Comment:
Damit der blaue focus border weggeht wenn man in das Input clickt:

```
&:focus {
    outline: none;
}
```

Reply:
guter tipp, gar nicht dran gedacht 👍